modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -393,3 +393,54 @@ spacewalk-services restart
 ====
 Increasing this value will cause {productname} Server to take longer to check if a minion is unreachable or unresponsive, causing the {productname} Server to be slower or less responsive overall.
 ====
+
+
+== Salt 3000 to Salt Bundle Migration
+
+
+=== Switch {sles} 12 or {centos} 7 minions (Salt 3000 EOL) to Salt Bundle.
+
+.Procedure: Switching with [literal]``util.mgr_switch_to_venv_minion`` state to [package]``venv-salt-minion``
+
+. Apply [literal]``util.mgr_switch_to_venv_minion`` with no pillar specified first.
+  This will result in the switch to [package]``venv-salt-minion`` with copying configuration files etc.
```

Comment:
```suggestion
  This will result in the switch to [package]``venv-salt-minion`` with copying configuration files and customization.
```
I hope we can use this wording to avoid the obscure "etc.".

Reply:
I believe this was meant to be "in etc" but I'm not absolutely sure. This was copied from https://documentation.suse.com/suma/4.3/en/suse-manager/client-configuration/contact-methods-saltbundle.html#_switching_from_salt_minion_to_salt_bundle
@agraul it this correct?

Reply:
I don't know. That wording was introduced in https://github.com/uyuni-project/uyuni-docs/pull/1458/files as far as I can tell. No one commented on it, so I can't speak to the authors intention.

Reply:
Thanks for digging it out (I was involved, but do not remember any details).  Maybe, @meaksh know whether the "/etc" directory was meant or other things ("etc.") -- if the latter, for example what?  Maybe, we can simply delete the etc here?  Pablo, I hope you can help!

Reply:
looks fixed now.